data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd

```diff
@@ -309,14 +318,14 @@ ck <- filter(butte_rst, gear_id == "diversion fyke trap 1")
 filter(butte_rst, is.na(fork_length), count > 0) %>% glimpse
 filter(butte_rst, is.na(mark_code)) %>% tally()
 
-butte_rst_clean <- butte_rst %>%
-  select(date, station, dead, count, fork_length, weight, mark_code, lifestage) %>%
+butte_rst_clean_pre_2015 <- butte_rst %>%
+  select(date, station, dead, count, fork_length, weight, mark_code, lifestage, gear_id) %>%
   rename(adipose_clipped = mark_code, 
          subsite = station) %>%
   mutate(stream = "butte creek",
          site = case_when(grepl("Okie Dam", subsite) ~ "okie dam",
                           T ~ tolower(subsite)),
-         subsite = tolower(subsite),
+         subsite = ifelse(gear_id == "diversion fyke trap 1", "okie fyke trap", tolower(site)),
```

Comment:
I think this should be: `subsite = ifelse(gear_id == "diversion fyke trap 1", "okie fyke trap", tolower(subsite)),` subsite instead of site because we want to retain okie dam 1 and okie dam 2, right?

Reply:
Fixed. Thanks for catching this